Homeworks/homework_11.py

```diff
@@ -0,0 +1,74 @@
+"""
+Homework 11.1.
+
+Ваша команда та ви розробляєте систему входу для веб-додатка,
+і вам потрібно реалізувати тести на функцію
+для логування подій в системі входу.
+Дано функцію, напишіть набір тестів для неї.
+"""
+
+import logging
+
+import pytest
+
+
+def read_file(log_level, message):
+    """
+    Read file.
+
+    :param log_level: logging level
+    :param message: message text
+    """
+    expected_result = f'{log_level} - {message}'
+    with open('homework11_login_system.log', 'r') as file:
+        text_in_file = file.readlines()
+        len_text_file = len(text_in_file)
+        actual_result = text_in_file[len_text_file - 1]
+        actual_result = ' - '.join(actual_result.split(' - ')[1:])
+        assert actual_result[:-1] == expected_result
+
+
+@pytest.mark.parametrize('username, status', [
+    ('Alice', 'success'),
+    ('Mykola', 'expired'),
+    ('Taras', 'failed'),
+])
+def test_log_event(username: str, status: str):
```

Comment:
Please split this test case into several test cases

Reply:
https://github.com/AnkaMikola/AQA_Hillel_git/pull/35